### ISSUE TITLE
the language of the slovak login page

### ISSUE THREAD

**Issue by mediapk** [lmsace/eguru#2]
Hello, when I change the language into Slovak, the login window in theme eGuru shows firm text: "Login into your account" instead of Slovak translation "Prihláste sa do vášho účtu" It is the same in other languages.

**Comment by ouzun** [comment]
same as mine. i use Turkish language and it shows english Login into your account

**Comment by mediapk** [comment]
Yes, It is a bug in the code. The best way is ask LMSACE for repair.

Pavol

2018-03-15 12:53 GMT+01:00 ouzun <notifications@github.com>:

**Comment by muga42** [comment]
I solved the problem by adding this code in the moodle settings "HTML additional".

`<script>
var Login = document.querySelectorAll("h2");
if (Login[1].innerHTML == "Login into your account")
{
Login[1].innerHTML = "Your login message";
}
</script>`

Put your message in `"Your login message"`. It's not an ideal solution, but will work for what you need.

**Comment by atronat** [comment]
To solve this problem you need to fix the [string 126](https://github.com/lmsace/eguru/blob/56cbf34a13ac900f0c8e82dfa8a242989d74be74/templates/login_form.mustache#L126) in the file  eguru/templates/login_form.mustache
String must be like this:
&lt;h2&gt;{{#str}}loginheader,theme_eguru{{/str}}&lt;/h2&gt;
